flare_portal/experiments/templates/experiments/researcher_list.html

```diff
@@ -0,0 +1,90 @@
+{% extends "base.html" %}
+
+{% load experiments_tags widget_tweaks %}
+
+{% block title %}Researchers - {{ project.name }}{% endblock title %}
+
+{% block content %}
+<div class="container">
+    <div class="row">
+        <div class="col-lg-4">
+            {% include "experiments/includes/project_sidebar.html" %}
+        </div>
+        <div class="col-lg-8">
+            <div class="card" x-data="{ forms: [], showPicker: false, startIndex: {{ researchers.count }} }">
+                <div class="card-header">
+                    <div class="d-flex align-items-baseline">
+                        <h3 class="card-title">Researchers</h3>
+                        <p class="card-subtitle m-0 ml-2">{{ researchers|length }} researcher{{ researchers|length|pluralize }}</p>
+                    </div>
+                    <div class="card-options">
+                        <button class="btn btn-sm btn-primary ml-4" form="add-researcher-form" type="submit">Save changes</button>
```

Comment:
issue (ux,non-blocking): should be Title Case? Same below.

```suggestion
                        <button class="btn btn-sm btn-primary ml-4" form="add-researcher-form" type="submit">Save Changes</button>
```

Reply:
quesstion: shouldn’t this also be wrapped in a `x-if="showPicker == true"`?

Reply:
Ah yes, Thanks!